mods/default/functions.lua

```diff
@@ -325,47 +325,69 @@ default.after_place_leaves = function(pos, placer, itemstack, pointed_thing)
 	end
 end
 
--- Leafdecay ABM
-
-minetest.register_abm({
-	label = "Leaf decay",
-	nodenames = {"group:leafdecay"},
-	neighbors = {"air"},
-	interval = 2,
-	chance = 10,
-	catch_up = false,
-
-	action = function(pos, node, _, _)
-		-- Check if leaf is placed
-		if node.param2 ~= 0 then
-			return
-		end
+-- Leafdecay
+default.leafdecay_speed = 15
+local function leafdecay_after_destruct(pos, oldnode, def)
+	local speed_min = (default.leafdecay_speed * 0.75) * 10
+	local speed_max = (default.leafdecay_speed / 0.75) * 10
+	for _, v in pairs(minetest.find_nodes_in_area(vector.subtract(pos, def.leaves_radius),
+			vector.add(pos, def.leaves_radius), def.leaves)) do
+		minetest.get_node_timer(v):start(math.random(speed_min) / 10,
```

Comment:
Nevermind my comment, i could not find docs of a 2-argument `start` so assumed this was an error and was meant to be `set`.